R/EstimationDataFromComparativeEffectivenessModel.R

```diff
@@ -0,0 +1,114 @@
+#' 
+#' @param connectionDetails    (optional) An object of type \code{connectionDetails} as created using the
+#'                             \code{\link[DatabaseConnector]{createConnectionDetails}} function in the
+#'                             DatabaseConnector package.
+#' @param preferenceScoreDist  (optional) A R-dataFrame object with fields named in camelCase containing 
+#'                             data from preference_score_dist table in comparative effectiveness data model.
+#' @param exposureOfInterest   (optional) A R-dataFrame object with fields named in camelCase containing
+#'                             data from exposure_of_interest table in comparative effectiveness data model.
+#' @param databaseId           A short string for identifying the database for query from comparative effectiveness
+#'                             data model (e.g. 'Synpuf')
+#' @param targetId             An integer identifying a cohort in exposure_id field of exposure_of_interest
+#'                             of OHDSI comparative effectiveness data model
+#' @param comparatorId         An integer identifying a cohort in exposure_id field of exposure_of_interest
+#'                             of OHDSI comparative effectiveness data model
+#' @param analysisId           An integer identifying the analysis_id field in cohort_method_analysis table
+#'                             of OHDSI comparative effectiveness data model
+#' @param formatName           (optional) Should the cohort name be formatted to remove prefixes and underscores? Default = FALSE
+#'
+#' @return                     tibble data frame
+#' 
+#' @examples                    SkeletonComparativeEffectStudy::getDataForPreferenceScorePlot(preferenceScoreDist = preferenceScoreDist, 
+#'                                                                                           exposureOfInterest = exposureOfInterest,
+#'                                                                                           targetId = 100,
+#'                                                                                           comparatorId = 200,
+#'                                                                                           databaseId = 'HCUP data',
+#'                                                                                           analysisId = 2)
+#' @export
+#'
+getDataForPreferenceScorePlot <- function(connection = NULL,
+                                          preferenceScoreDist = NULL,
+                                          exposureOfInterest = NULL,
+                                          targetId,
+                                          comparatorId,
+                                          outcomeId,
+                                          databaseId,
+                                          analysisId
+) {
+
+  
+  errorMessage <- checkmate::makeAssertCollection()
+  checkmate::assertInt(targetId, add = errorMessage)
```

Comment:
Add checkmate to imports in DESCRIPTION. Did you run R check locally? This would have been flagged. Please don't submit pull requests with R check warnings still present.